.github/workflows/ci-proto.yml

```diff
@@ -22,6 +22,16 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
```

Comment:
arduino?! looks very suspicious.

Reply:
I can switch to naive installation of protoc, but I don't see a problem with this 
if you prefer to switch, please open a ticket and assign it to me